src/test/java/org/owasp/wrongsecrets/challenges/docker/Challenge29Test.java

```diff
@@ -0,0 +1,23 @@
+package org.owasp.wrongsecrets.challenges.docker;
+
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+import org.owasp.wrongsecrets.ScoreCard;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+@ExtendWith(MockitoExtension.class)
+class Challenge29Test {
+
+    @Mock
+    private ScoreCard scoreCard;
+    @Test
+    public void testGetMyString() {
```

Comment:
Can you please expand these tests and the Challengecontrolertests once the feedback items above are done?